src/crypto/store/indexeddb-crypto-store-backend.js

```diff
@@ -257,6 +257,38 @@ export class Backend {
         };
         return promiseifyTxn(txn);
     }
+
+    /**
+     * Load the end to end account for the logged-in user. Once the account
+     * is retrieved, the given function is executed and passed the base64
+     * encoded account string and a method for saving the account string
+     * back to the database. This allows the account to be read and writen
+     * atomically.
+     * @param {func} func Function called with the account data and a save function
```

Comment:
```
@param {function(string, function())} func ....
```
would be the correct annotation here I think

Reply:
done